packages/web/src/components/inbox-unavailable-modal/InboxUnavailableModal.tsx

```diff
@@ -110,7 +111,8 @@ export const InboxUnavailableModal = ({
       dispatch(createChat({ userIds: [user.user_id] }))
       onClose()
     } else {
-      // TODO: Link to blog post
+      window.open(CHAT_BLOG_POST_URL, '__blank')
```

Comment:
`_blank`?

Reply:
indeed, fixed in another place as well. copy and paste error